lib/ae_mdw/channels.ex

```diff
@@ -102,6 +135,39 @@ defmodule AeMdw.Channels do
     end
   end
 
+  defp build_nested_streamer(state, updates, range, cursor) do
```

Comment:
This is a naming nit pick as the output data comes from the list that was already fully fetched and is not transformed in a lazy way. Maybe sth like `build_updates_filter`?